addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/PersistentXssScanRule.java

```diff
@@ -680,6 +680,29 @@ public void scan(HttpMessage sourceMsg, String param, String value) {
         }
     }
 
+    @Override
+    public List<Alert> getExampleAlerts() {
+        return List.of(
+                buildAlert(
+                                "https://example.com/comments",
+                                "comment",
+                                "<script>alert('XSS');</script>",
+                                'P',
+                                "HTTP/1.1 500 Internal Server Error")
+                        .build());
+    }
+
+    private AlertBuilder buildAlert(
```

Comment:
The idea is to have the existing `newAlert()` calls to use this new method (adjusted as needed), to avoid duplication and ensure the example alerts match the raised alerts.